runtime/tests/CMakeLists.txt

```diff
@@ -43,7 +43,9 @@ add_subdirectory(port)
 add_subdirectory(props)
 add_subdirectory(redirector)
 add_subdirectory(shared)
-# add_subdirectory(shared_service)
+if(J9VM_OPT_SHARED_SERVICETEST)
```

Comment:
`J9VM_OPT_SHARED_SERVICETEST` doesn't appear to ever be defined anywhere.
Also, `win_x86.spec` says `<flag id="module_shared_servicetest" value="true"/>`, so I don't think the test needs to be excluded for win32 (or any other platform).

Reply:
Looks like there were unstaged changes that got left behind. This should be `J9VM_MODULE_SHARED_SERVCETEST`. Added if-gating as we only want this test to run on win32 (it appears to be the only spec file with `module_shared_servicetest` set)

Reply:
As it is for win32 only, I think it makes more sense to use the condition `OMR_OS_WINDOWS AND NOT OMR_ENV_DATA64` rather than introduce yet another cmake cache variable (`J9VM_MODULE_SHARED_SERVCETEST`).

Reply:
@dnakamura I think removing `J9VM_MODULE_SHARED_SERVICETEST` is the only change holding this up.